src/modules/app/AppComponent.js

```diff
@@ -148,10 +132,12 @@ export class AppComponent extends Component {
               <LinkContainer to={'/search/objects'}>
                 <NavItem><FontAwesome name="search" style={{ fontSize: '1.3em', height: 25 }} /></NavItem>
               </LinkContainer>
-              <LinkContainer className="toggleScanner" to={'/scan'} active={this.props.scanner.enabled} onClick={(e) => {
-                e.preventDefault();
-                this.props.toggleEnabled();
-              }}>
+              <LinkContainer
+                className="toggleScanner"
+                to={'/scan'}
+                disabled={this.props.count.value > 0}
+                active={this.props.count.value > 0}
```

Comment:
This button is at the moment just a half assed fix for the button, for which there no longer is any toggle functionality for. The idea behind disabled is to avoid clicking on it, but it could be always disabled for that matter. The active property makes the menu item appear selected.

But all in all, this button should not be here. We should add a button on each page that is context aware.